linux/buildSrc/checkstyle.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
```

Comment:
Not added to the build.gradle?

Reply:
It is, see `linux/buildSrc/build.gradle` (L22-L27). It's only there for the plug-in sources. Or you do mean something else?

Reply:
I missed it, my bad!